.prettierrc

```diff
@@ -1,6 +1,7 @@
 {
   "tabWidth": 2,
   "useTabs": false,
+  "singleQuote": true,
```

Comment:
Any reason to switch to single quotes? Looks like this has introduced several linting errors.

Reply:
It's just that the most of the repositories I contribute to in typescript use this rule so I added it (It can be removed). Anyway I know I will need to update the tests to not have any lint errors (I've just postpoined it so far because I wanted the other PR for the reclaimer facet to be merged)